scripts/generate-changelog-entry.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Could we include a make command to invoke the script?

Reply:
Done!